src/main.ts

```diff
@@ -1,10 +1,13 @@
 import config from './config'
 import Express from 'express'
 import { buildAggregates, connectMongoose, createServer } from './server'
+import { cryptoWaitReady } from '@polkadot/util-crypto'
 
 const main = async () => {
   config.loadConfig()
 
+  await wrapTask('Initializing WASM bridge for @polkadot/util-crypto', cryptoWaitReady)
```

Comment:
I'm getting this on startup:

```
Error: The WASM interface has not been initialized. Ensure that you wait for the initialization Promise with waitReady() from @polkadot/wasm-crypto (or cryptoWaitReady() from @polkadot/util-crypto) before attempting to use WASM-only interfaces.
```

Does it work for you?

Reply:
Fixed